Frameworks/UIKit/UIControl.mm

```diff
@@ -221,7 +221,7 @@ - (NSSet*)allTargets {
 */
 - (void)setEnabled:(BOOL)enabled {
     // only update the value and then relayout if value actually changed
-    if ((_curState & UIControlStateDisabled) != enabled) {
+    if ((_curState & UIControlStateDisabled) != !enabled) {
```

Comment:
this just follows precedence from elsewhere.  it's unclear to me why this isn't safe?

---
In reply to: [99914946](https://github.com/Microsoft/WinObjC/pull/1920#discussion_r99914946) [](ancestors = 99914946)